src/Extensions/InternetExtension.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Xefi\Faker\Extensions;
+
+class InternetExtension extends Extension
+{
+    protected $tld = ['com', 'biz', 'info', 'net', 'org', 'edu', 'gov', 'mil', 'co', 'io'];
+
+    public function sdl(): string
+    {
+        return $this->randomizer->getBytesFromString(
+            implode(range('a', 'z')),
+            $this->randomizer->getInt(5, 10)
+        );
+    }
+
+    public function tld(): string
+    {
+        return $this->pickArrayRandomElement($this->tld);
+    }
+
+    public function domain(): string
+    {
+        return sprintf('%s.%s', $this->sdl(), $this->tld());
+    }
+
+    public function ipv4(): string
+    {
+        return long2ip($this->randomizer->getInt(0, 1) === 0 ? $this->randomizer->getInt(-2147483648, -2) : $this->randomizer->getInt(16777216, 2147483647));
+    }
+
+    public function ipv6(): string
+    {
+        $res = [];
+
+        for ($i = 0; $i < 8; $i++) {
+            $res[] = dechex($this->randomizer->getInt(0, 65535));
```

Comment:
Do you prefer this way to ensure compatibility, or to use `inet_ntop` with a randomizer, which is the prefered way of using IPv6 when PHP is compiled with IPv6 ?

Reply:
I did have a look at this but i'm not familiar enough with this function and documentation didn't help me on that, but this isn't a reason, will need to have a look